docs/ar/index.md

```diff
@@ -0,0 +1,288 @@
+---
+layout: default
+title: DEEP LEARNING
+author: Alfredo Canziani
+lang-ref: home
+---
```

Comment:
`lang: ar` is missing. My script does not know what language this is supposed to be.

Reply:
See for example the Turkish version:

```
atcold@AlfMAC2 ~/Work/GitHub/pDL/docs [master*]
$ head tr/index.md 
---
layout: default
title: DERIN ÖĞRENME
author: Alfredo Canziani
lang-ref: home
lang: tr
translation-date: 1 July 2020
translator: Murat Ekici
---
```

Reply:
Done.